lib/MetaCPAN/Document/File/Set.pm

```diff
@@ -499,28 +499,47 @@ sub autocomplete_using_suggester {
         }
         );
 
-    my @docs
-        = map { $_->{text} } @{ $suggestions->{documentation}[0]{options} };
+    my %docs;
+    for my $d ( @{ $suggestions->{documentation}[0]{options} } ) {
```

Comment:
What is `$d`?  It's not immediately clear to me.  Maybe a more descriptive name?